### PR TITLE
fix(tool): add asserts to changelog code to make it less fragile.

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: build-system/tasks/changelog.js [hunk 1]
```diff
@@ -69,6 +70,7 @@ function getGitMetadata() {
 }
 
 function submitReleaseNotes(version, changelog) {
+  assert(typeof version == 'number', 'version should be a number.');
```
Comment (cramforce) [inline build-system/tasks/changelog.js]: Output %s, version

File: build-system/tasks/changelog.js [hunk 2]
```diff
@@ -169,6 +171,7 @@ function fetchGithubMetadata(ids) {
 function getPullRequestTitle(prOption) {
   return request(prOption).then(function(res) {
     var body = JSON.parse(res.body);
+    assert(typeof body.url == 'string', 'should have url string.');
```
Comment (cramforce) [inline build-system/tasks/changelog.js]: Output %s, body

Comment (cramforce) [inline build-system/tasks/changelog.js]: When you fail, output res.body

File: build-system/tasks/changelog.js (hunk 1 above)
Comment (erwinmombay) [inline build-system/tasks/changelog.js]: done.

File: build-system/tasks/changelog.js (hunk 2 above)
Comment (erwinmombay) [inline build-system/tasks/changelog.js]: done.

File: build-system/tasks/changelog.js [hunk 3]
```diff
@@ -63,12 +66,14 @@ function getGitMetadata() {
...
       .then(buildChangelog.bind(null, gitMetadata))
       .then(function(gitMetadata) {
+        util.log(util.colors.blue('\n' + gitMetadata.changelog));
         return submitReleaseNotes(version, gitMetadata.changelog);
       })
       .catch(errHandler);
 }
 
 function submitReleaseNotes(version, changelog) {
+  assert(typeof version == 'number', 'version should be a number. ' + version);
```
Comment (cramforce) [inline build-system/tasks/changelog.js]: Supports %s interpolation

**Comment by cramforce** [comment]
LGTM with one comment

**Pull request merged**